nilearn/plotting/surf_plotting.py

```diff
@@ -483,7 +483,7 @@ def plot_surf_roi(surf_mesh, roi_map, bg_map=None,
                          'value for each vertex, or a list of Numpy arrays, '
                          'one array per ROI which contains indices of all '
                          'vertices included in that ROI')
-
+    vmin, vmax = np.min(roi_map), 1 + np.max(roi_map)
```

Comment:
Just for a point to be noted: Why is this needed ?

Reply:
otherwise if there is only one value (e.g. plotting a mask) it does not appear on the plot with some colormaps. for example try removing this line and running the plot_surf_stat_map.py example: the seed plot is empty

Reply:
But, we didn't caught this in the example before ?